src/backend/libc/fs/types.rs

```diff
@@ -798,7 +806,7 @@ bitflags! {
         const NOATIME = libc::ST_NOATIME as u64;
 
         /// `ST_NODEV`
-        #[cfg(any(target_os = "android", target_os = "emscripten", target_os = "fuchsia", target_os = "linux"))]
+        #[cfg(any(target_os = "android", target_os = "emscripten", target_os = "fuchsia", target_os = "linux", target_os = "aix"))]
```

Comment:
Please put the `target_os = "aix"` at the beginning of the `any` list, to keep it in alphabetical order.